wurst/_handles/Unit.wurst

```diff
@@ -283,6 +292,18 @@ public function unit.isInRange(vec2 pos, real distance) returns bool
 public function unit.isEnemyOf(player whichPlayer) returns bool
 	return IsUnitEnemy(this, whichPlayer)
 
+/** Checks whether two units are enemies */
+public function unit.isEnemyOf(unit u) returns bool
+	return this != null and u != null and IsUnitEnemy(this, u.getOwner())
```

Comment:
You don't need to check `!= null` to get proper results. Warcraft handles such cases in his own correct way. And even if it does not, the check is not supposed to be here for consistency reasons.

Reply:
> Warcraft handles such cases in his own correct way.

Ha! No it doesn't @Trokkin 

wc3 is awful on null checks for unit handles

But, maybe it should be removed, as map author's responsibility. I am not sure. @Frotty ?